src/components/tool/guide/EresearchJob/EresearchJob.js

```diff
@@ -93,6 +93,30 @@ const getConfigGroups = (config, onConfigChange = () => {
                 selected: (config) => config?.nodes > 0,
             }
         },
+        "wallTime": () => {
+            return {
+                element: (key, selected) => <ConfigDuration
+                    key={key}
+                    title="Walltime"
+                    description="The estimated total time for the job."
+                    selected={selected}
+                    onChange={onChange("wallTime")}
+                    value={config?.wallTime}
+                    inputPropsHour={{
+                        min: 1,
```

Comment:
Min hours should be 0

Reply:
If Min hours is 0 and min minute is 0, the job will likely be unsuccessful?

Reply:
Almost certainly, yes. Although surprisingly enough it will start and run for less than a minute. A minimum of 1 minute (and `0` when `hours > 0`) is probably fine, but we probably don't want a strict maximum.